snippets/capabilities-table.adoc

```diff
@@ -14,12 +14,14 @@ The following table describes the `baselineCapabilitySet` values.
 |Specify when you want the capabilities recommended in {product-title} 4.11 and not automatically enable capabilities, which might be introduced in later versions. The capabilities recommended in {product-title} 4.11 are `baremetal`, `marketplace`, and `openshift-samples`.
 
 |`v4.12`
-|Specify when you want the capabilities recommended in {product-title} 4.12 and not automatically enable capabilities, which might be introduced in later versions. The capabilities recommended in {product-title} 4.12 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage` and `CSISnapshot`. 
+|Specify when you want the capabilities recommended in {product-title} 4.12 and not automatically enable capabilities, which might be introduced in later versions. The capabilities recommended in {product-title} 4.12 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage` and `CSISnapshot`.
 
 |`v4.13`
-|Specify when you want the capabilities recommended in {product-title} 4.13 and not automatically enable capabilities, which might be introduced in later versions. The capabilities recommended in {product-title} 4.12 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage`, `CSISnapshot` and `NodeTuning`. 
+|Specify when you want the capabilities recommended in {product-title} 4.13 and not automatically enable capabilities, which might be introduced in later versions. The capabilities recommended in {product-title} 4.13 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage`, `CSISnapshot` and `NodeTuning`.
+
+// TODO: Add `Build` and `DeploymentConfig` to the list for 4.14.
```

Comment:
@bscott-rh Here's the PR that adds to the cluster capabilities page, if you can PTAL to make sure it looks okay to you. And here's a reminder whenever you add the v4.14 link, to add `Build` and `DeploymentConfig` please.

Reply:
The changes look good to me. Thank you